src/main/resources/templates/db_scripts/populateEffectiveLocationForExistingItems.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE ${myuniversity}_${mymodule}.item AS it
+-- Since holdings_record.permanentLocationId and item.holdingsRecordId are required there can not be a NULL value
+SET	jsonb = JSONB_SET(it.jsonb,
+			'{effectiveLocationId}',
+			COALESCE(it.jsonb->'temporaryLocationId', it.jsonb->'permanentLocationId',
+			hr.jsonb->'temporaryLocationId', hr.jsonb->'permanentLocationId')
+)
+FROM ${myuniversity}_${mymodule}.holdings_record AS hr
+WHERE it.jsonb->'effectiveLocationId' IS NULL AND hr.id = it.holdingsrecordid;
```

Comment:
I've added unit tests for populateEffectiveLocationForExistingItems.sql. Two of them fail. Please fix.

Reply:
Hi @julianladisch, thanks! I don't think that both tests are correct. 

The migration script will be executed only when effectiveLocationId is not set for an item, otherwise we assume that the item already got handled by triggers, so nothing to do with it.

Am I missing something?